packages/lms/src/app.html

```diff
@@ -2,12 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%sveltekit.assets%/favicon.png" />
+    <link rel="icon" type="image/x-icon" href="favicon.ico" />
```

Comment:
This is intentional, it is overwritten by the build process. You may not have the asset in /static

Reply:
Reverted the change